bedrock/firefox/templates/firefox/welcome/base.html

```diff
@@ -20,8 +20,9 @@
   <header class="c-page-header">
     <div class="mzp-l-content c-page-header-inner">
       {% block shoulder_logo %}
-        <img src="{{ static('protocol/img/logos/firefox/browser/logo-word-hor.svg') }}" alt="Firefox" width="216" height="40" class="c-header-logo">
+        <img src="{{ static('img/firefox/welcome/firefox-wordmark-hor.svg') }}" alt="Firefox" width="130" height="40" class="c-header-logo">
```

Comment:
Or even better, there's one already in img/logos:

```suggestion
        <img src="{{ static('img/logos/firefox/logo-word-hor.svg') }}" alt="Firefox" width="120" height="40" class="c-header-logo">
```

(that also has the correct dimensions iirc)

Reply:
Ah wow thanks for finding this! I'll implement this logo for pages 16-18 dark/light mode as well 🌞🌚